### ISSUE TITLE
Could not detect Docker VM

### ISSUE THREAD

**Issue by thebillkidy** [kitematic/kitematic#160]
When booting the application it gets stuck on Detecting Docker VM.

This is what I get:
```
[4133:0119/162102:INFO:CONSOLE(175)] "Kitematic started.", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (175)
[4133:0119/162102:INFO:CONSOLE(175)] "Setting host IP to: null", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (175)
Checking for update...
[4133:0119/162102:INFO:CONSOLE(173)] "Performing step 0", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (173)
[4133:0119/162102:INFO:CONSOLE(173)] "Performing step 1", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (173)
[4133:0119/162102:INFO:CONSOLE(173)] "Performing step 2", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (173)
[4133:0119/162103:INFO:CONSOLE(173)] "Performing step 3", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (173)
Update not available.
[4133:0119/162111:INFO:CONSOLE(173)] "Performing step 4", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (173)
[4133:0119/162111:INFO:CONSOLE(173)] "Setting host IP to: 192.168.59.103", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (173)
[4133:0119/162111:INFO:CONSOLE(173)] "Setup finished.", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (173)
[4133:0119/162111:INFO:CONSOLE(157)] "Uncaught TypeError: Cannot read property 'listImages' of null", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (157)
2015-01-19T16:22:01.336+0100 [clientcursormon] mem (MB) res:51 virt:2851
2015-01-19T16:22:01.336+0100 [clientcursormon]  mapped (incl journal view):320
2015-01-19T16:22:01.336+0100 [clientcursormon]  connections:5
```

seeing that: ```[4133:0119/162111:INFO:CONSOLE(157)] "Uncaught TypeError: Cannot read property 'listImages' of null", source: http://localhost:63376/a141ac2bc620e813d10b2de9017e7830bea956a9.js (157)``` I think there is a null variable somewhere causing it not to switch the screen. Probably something minor, but still won't show me anything.

**Comment by jeffdm** [comment]
Hey there-  thanks for creating an issue.

This is the line where it fails: https://github.com/kitematic/kitematic/blob/f6f67ba647b9b87e7b1465c84c7ce59bddc588a9/meteor/client/lib/imageutil.js#L233

It looks like a docker client couldn't be created (and therefore returned `null`). I'm looking into why this is happening now

**Comment by dvbportal** [comment]
Same here. I have been upgrading from 0.2.4 to 0.4.5. The old version was running and I had several images/apps installed.

**Comment by southwolf** [comment]
Version 0.5.10, still not able to detect boot2docker v1.50 vm. Kitematic created a vm called 'dev'.

**Comment by jeffdm** [comment]
This should be fixed with 0.5.10.

That said, note that Kitematic won't automatically pick up an existing boot2docker setup (we recently switched to using [docker-machine](https://github.com/docker/machine).
